lms/static/styles/components/_Dialog.scss

```diff
@@ -70,3 +72,9 @@
     }
   }
 }
+
+.Dialog__dimension {
+  /* See https://css-tricks.com/flexbox-truncated-text/ */
+  min-height: 0;
+  width: 75%;
```

Comment:
I understand the linked article, but I'm not clear on what you are trying to do here. Can you clarify?